ThermofluidStream/Boundaries/Sink.mo

```diff
@@ -13,6 +13,8 @@ the outlet the sink is connected to.
     annotation(Dialog(enable = not pressureFromInput));
   parameter Utilities.Units.Inertance L=dropOfCommons.L "Inertance of pressure"
     annotation (Dialog(tab="Advanced"));
+  parameter SI.MassFlowRate m_flow_assert(max=0) = -dropOfCommons.m_flow_reg "Assertion threshold for negative massflows"
```

Comment:
@nieweber The existing `PartialVolume` has this:
```modelica
  parameter SI.MassFlowRate m_flow_assert(max=0) = -dropOfCommons.m_flow_reg "Assertion threshold for negative massflows"
```
Per your suggestions, `Source` and `Sink` will be inconsistent. If we instead change `PartialVolume`, then it would not be backward compatible.

Please confirm that you want it to be inconsistent or let me know what you would like instead and I'll make the change per your suggestions.

Reply:
Oh yes thank you, you are completely right. Sorry, I didn't cross-check with the assert in the `PartialVolume`. The components should indeed be consistent so we will discuss, if we want to include it in the next minor release (1.1.0) and keep the negative sign or if we shift this change to the next major release and adjust the `PartialVolume` assert.
I will let you know what we would like to do.